root/mbid/NotFound.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import NotFound from '../components/NotFound';
+
+const MbidNotFound = ({isGuid, mbid}: {+isGuid: boolean, +mbid: string}) => (
+  <NotFound title={isGuid ? l('MBID Not Found') : l('Invalid MBID')}>
+    <p>
+      {isGuid ? (
+        texp.l(
+          `No MusicBrainz entities match the MBID {mbid}.
+           Either it’s incorrect, or the entity has since been deleted.`,
+          {mbid: mbid},
+        )
+      ) : (
```

Comment:
GUID is used in MusicBrainz not only for entities : alternative release, release status, release type, media attribute type, media attribute type allowed value, and so on, that might be visible from various source (json-ld, web service, Picard-tagged files, potentially any 3rd-party using MB data) and thus legitimately looked at through this new `/mbid/` URL.

Please link to available documentation and shortly mention this other possible case.

Reply:
What about something like: 
`No MusicBrainz entities match the MBID {mbid}. Either it’s incorrect, it was for an entity that has since been deleted, or it is for something else than an entity (for example, a relationship type).`

The only link that seems totally relevant is https://musicbrainz.org/doc/MusicBrainz_Identifier but that's pretty technical and not very good as a doc. We might want to improve those docs eventually, but is it a good idea to link to that in the meantime?

Reply:
Yes, plus https://musicbrainz.org/doc/MusicBrainz_Entity cause entity is a developer term.

Reply:
Ok, added a bit more info to the message as suggested.